core/templates/services/rte-helper.service.ts

```diff
@@ -61,6 +61,7 @@ export class RteHelperService {
   openCustomizationModal(
       customizationArgSpecs: CustomizationArgsSpecsType,
       attrsCustomizationArgsDict: CustomizationArgsForRteType,
+      componentId: CustomizationComponentId,
```

Comment:
This is not a "customization component id". It is an "RteComponentId". The customizationArgSpecs customize the component, but the component ID identifies its type uniquely.

Please fix so that we maintain conceptual clarity within the codebase -- thanks.